common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -391,12 +403,29 @@ def get_progress(self):
                 return None
         return None
 
+    def show_progress(self, **kwargs):
+        """
+        Return (score, total) to be displayed to the learner.
+        """
+        if 'progress' in kwargs:
```

Comment:
Actually, wouldn't it make more sense to just embed the logic of making the score None in get_progress itself? When do you want get_progress to return a score when self.correctness_available() is False?